packages/fx-core/src/plugins/resource/frontend/index.ts

```diff
@@ -55,10 +57,14 @@ export class FrontendPlugin implements Plugin {
   }
 
   public async provision(ctx: PluginContext): Promise<TeamsFxResult> {
-    FrontendPlugin.setContext(ctx);
-    return this.runWithErrorHandling(ctx, TelemetryEvent.Provision, () =>
-      this.frontendPluginImpl.provision(ctx)
-    );
+    if (isArmSupportEnabled()) {
```

Comment:
What if `isArmSupportEnabled` throws error? Do we have readable error message? Do we send telemetry?

Reply:
isArmSupportEnabled  is an env flag. 
Actually, we just skip provision step, and there will be no error.
If we use arm, we will not do provision in plugin.

Reply:
I think the provision step in frontend plugin will be removed eventually. The env flag is temporary solution. Is it right?

Reply:
Yes.